code/game/machinery/_machinery.dm

```diff
@@ -204,6 +204,22 @@ Class Procs:
 			L.update_mobility()
 	occupant = null
 
+/**
+ * Places passed object in to users hand
+ *
+ * Places the passed object in to the users hand if they are ajacent.
+ * If the user is not ajacent then place the object on top of the machine.
+ *
+ * Vars:
+ * * object (obj) The object to be moved in to the users hand.
+ * * user (mob/living) The user to recive the object
+ */
+/obj/machinery/proc/hand_object(obj/object, mob/living/user)
```

Comment:
```suggestion
/obj/machinery/proc/try_put_in_hands(obj/object, mob/living/user)
```
or anything more clear than `hand_object`

Reply:
Did a better name, hand_object that is just magical ha